sdk/src/main/java/com/hedera/hashgraph/sdk/FreezeTransaction.java

```diff
@@ -37,7 +45,11 @@ public FreezeTransaction() {
     }
 
     public Instant getStartTime() {
-        return Instant.from(OffsetTime.of(startHour, startMinute, 0, 0, ZoneOffset.UTC));
+        if(startTime != null) {
```

Comment:
I feel like you should simply do `return startTime != null ? startTime : Instant.EPOCH` We shouldn't be using the other fields anymore anyways, right?